### DISCUSSION TITLE
Globally Changing Default Sizes

### DISCUSSION THREAD

**Discussion by rcronin** [mantinedev/mantine#2140] [Q&A]
Hello,

Is there a way to globally change the default sizes? For example, fonts, buttons, padding, etc?

**Comment by rtivital** [comment]
Yes, see theming and styles section in documentation.

**Reply by telmen** [reply]
hi @rtivital, it's not clear how to do that. for example, how do I change button radiuses per size?

**Reply by rtivital** [reply]
https://mantine.dev/theming/mantine-provider/#styles-on-mantineprovider
